templates/common/_base/files/mtu-migration-conf.yaml

```diff
@@ -0,0 +1,32 @@
+{{ $target_mtu := 0 -}}
+{{ $current_mtu := 0 -}}
+{{ $cni_target_mtu := 0 -}}
+{{ if .Network }}{{ if .Network.MTUMigration -}}
+{{ if .Network.MTUMigration.Machine -}}
+{{ if .Network.MTUMigration.Machine.To -}}
+{{ $target_mtu = .Network.MTUMigration.Machine.To -}}
+{{ end -}}
+{{ if .Network.MTUMigration.Machine.From -}}
+{{ $current_mtu = .Network.MTUMigration.Machine.From -}}
+{{ end -}}
+{{ end -}}
+{{ if .Network.MTUMigration.Network }}{{ if .Network.MTUMigration.Network.To -}}
+{{ $cni_target_mtu = .Network.MTUMigration.Network.To -}}
+{{ end }}{{ end -}}
+{{ end }}{{ end -}}
+{{ if or $target_mtu $cni_target_mtu -}}
+mode: 0644
+path: "/etc/cno/mtu-migration/config"
+contents:
+  inline: |
+    NETWORK_TYPE={{ .NetworkType }}
+    {{ if $target_mtu -}}
+    TARGET_MTU={{ $target_mtu }}
+    {{ end -}}
+    {{ if $target_mtu -}}
+    CURRENT_MTU={{ $current_mtu }}
+    {{ end -}}
+    {{ if $cni_target_mtu -}}
+    CNI_TARGET_MTU={{ $cni_target_mtu }}
```

Comment:
maybe just comment that this is only for a use case where the host MTUs are not modified and just the CNI MTU is modified?